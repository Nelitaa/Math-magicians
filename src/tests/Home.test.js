import { render } from '@testing-library/react';
import Home from '../components/Home';

test('renders Home component', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});
