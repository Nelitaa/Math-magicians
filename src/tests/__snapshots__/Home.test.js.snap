// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home component 1`] = `
<DocumentFragment>
  <section>
    <h2>
      Welcome to our page!
    </h2>
    <p>
      Having a basic understanding of the development of technology over the course of history can help us to predict the potential growth of technology to come. Here, the professionals at Rechart take you through the history of the calculator, and how we came to have the ubiquitous devices we use every day.
    </p>
    <p>
      It is almost impossible for us to imagine mathematics without something as seemingly simple as a calculator. That being said, the calculators we know today were not invented until the 1970s, and the use of smartphones as calculators did not begin until at least the late nineties. But that doesn’t mean that mathematical tools were not available before the 20th century—there were many different computing machines created long before digital calculators and smartphones.
    </p>
    <p>
      The first tool created specifically for use in mathematical computations was the abacus, likely invented in Sumeria around 2500 B.C. The abacus was a table of successive columns with beads or stones representing a single unit, which could be used for addition or subtraction. Other cultures altered and refined the abacus; the Chinese, for example, put beads on wire within a bamboo frame to enhance ease of use. Unfortunately, the abacus was not very useful for multiplication or division, necessitating the invention of a new and more sophisticated device.
    </p>
  </section>
</DocumentFragment>
`;
